src/plugins/console/public/application/hooks/use_send_current_request/use_send_current_request.ts

```diff
@@ -28,7 +30,9 @@ export const useSendCurrentRequest = () => {
   return useCallback(async () => {
     try {
       const editor = registry.getInputEditor();
-      const requests = await editor.getRequestsInRange();
+      const variables = storage.get(StorageKeys.VARIABLES, []);
```

Comment:
Looks like these variables aren't being populated in the flyout. Can we populate them there, too?

![image](https://user-images.githubusercontent.com/1238659/179075183-d511b663-6729-4766-a5a6-9d29b6d16ab3.png)

I'm not sure if the `storage` service makes this possible but can we check if the `StorageKeys.VARIABLES` key is set and, if not, set the value with the `DEFAULT_VARIABLES`?